src/presentation/api/volunteer.ts

```diff
@@ -223,4 +225,47 @@ export class VolunteerAPI extends Controller {
         })
       );
   }
+
+  /**
+   * Post volunteer hours
+   *
+   */
+
+  @Post('hours')
+  // @Security('jwt', ['bookPermission'])
```

Comment:
```suggestion
  @Security('jwt')
```

Reply:
In that case, anyone that can log in can use this route (no specific authorization)